src/Akka.Cluster.Hosting/AkkaClusterHostingExtensions.cs

```diff
@@ -634,9 +640,10 @@ private static ExtractShardId ToExtractShardId(this IMessageExtractor self)
             async Task Resolver(ActorSystem system, IActorRegistry registry, IDependencyResolver resolver)
             {
                 var props = entityPropsFactory(system, registry, resolver);
-                var settings = ClusterShardingSettings.Create(system).WithRole(shardOptions.Role);
```

Comment:
This was a big bug - accidentally removed the code used to set up things like the state management system for sharding. This has been fixed.